sdk/compiler/daml-lf-ast/src/DA/Daml/LF/Ast/World.hs

```diff
@@ -124,23 +122,14 @@ data LookupError
   | LEAmbiguousInterfaceInstance !InterfaceInstanceHead
   deriving (Eq, Ord, Show)
 
-lookupPackageModules :: SelfOrImportedPackageId -> World -> Either LookupError (NM.NameMap Module)
```

Comment:
These changes were introduced in 512a521 and reverted in e25ec9e, could we add them back? We only needed to remove the warning, but these refactoring are helpful and would eliminate a potential bug further down the line.